Assets/TestScene/LightCameraTest.unity

```diff
@@ -1028,7 +1028,7 @@ Prefab:
       objectReference: {fileID: 0}
```

Comment:
What have you changed in the scene? There's nothing that could be done through prefab?